### ISSUE TITLE
Gitpod: Error :-( on first build

### ISSUE THREAD

**Issue by TrueMilli** [cleodora-forecasting/cleodora#171]
# First build on gitpod.

## Error in frontend:
![image](https://user-images.githubusercontent.com/61841994/212766421-3214c60b-8d16-4119-8f9c-eda311e1ccab.png)
## Forecasts can also not be added:
![image](https://user-images.githubusercontent.com/61841994/212768553-f8d11639-3102-458f-a733-69fbbe91f7b4.png)
## The Website is not served and I don't see how in the guide:
![image](https://user-images.githubusercontent.com/61841994/212768490-85d33293-0e5f-4934-b49c-3e83477412b2.png)
## I don't see a db:
![image](https://user-images.githubusercontent.com/61841994/212768595-7cbcf190-f780-42a4-8b71-79ec0e4d4331.png)
## npm start
![image](https://user-images.githubusercontent.com/61841994/212766462-3aa9edea-6b2f-4585-9b15-86199ee522ca.png)
## mage build
![image](https://user-images.githubusercontent.com/61841994/212768666-73640e4a-f5f8-433f-9503-5b4d2abd238e.png)
## go run .
![image](https://user-images.githubusercontent.com/61841994/212768945-27642d2f-85ca-4116-8e2c-eda668ae5227.png)

**Comment by omarkohl** [comment]
Thank you for your report! @all-contributors please add @TrueMilli for userTesting and bug.

**Comment by omarkohl** [comment]
* Concerning the frontend errors, the documentation was incomplete. Solved with 895fede . In short, the API URL is not http://localhost:8080 in GitPod (or GitHub Codespaces for that matter) and needs to be passed as an ENV variable (`REACT_APP_API_URL`) when starting the frontend.
* I also added #173 to improve the error message
* The path to the default database can be seen in your last screenshot `/home/gitpod/.local/share/cleosrv/cleosrv.db`
* How to start/build the website was not documented, but I now added it with commit 6fe97fa

Thank you for your thorough report!
